.eslintrc.json

```diff
@@ -0,0 +1,57 @@
+{
+    "env": {
+        "browser": true
+    },
+    "extends": "eslint:recommended",
+    "rules": {
+        "indent": [
+            "error",
+            2
+        ],
+        "linebreak-style": [
+            "error",
+            "windows"
+        ],
+        "quotes": [
+            "error",
+            "single"
+        ],
+        "semi": [
+            "error",
+            "always"
+        ],
+		"no-undefined": [
+			"error"
+		],
+		"no-unused-vars": [
+			"error"
+		],
+		"strict": [
+			"error",
+			"global"
+		],
+		"no-console": [
+			"error"
+		],
+		"comma-dangle": [
+			"error", 
+			"always"
+		],
+		 "max-len": [
+			"error", 
+			80, 
+			4, 
```

Comment:
This should match the indentation level you specified in the indentation rule (2).